generators/python-v2/codegen/src/ast/Uuid.ts

```diff
@@ -0,0 +1,42 @@
+import { AstNode } from "./core/AstNode";
+import { ClassInstantiation } from "./ClassInstantiation";
+import { Writer } from "./core/Writer";
+import { Reference } from "./Reference";
+import { MethodArgument } from "./MethodArgument";
+import { CodeBlock } from "./CodeBlock";
+
+export declare namespace UUID {
+    export interface Args {
+        value: string;
+    }
+}
+
+export class UUID extends AstNode {
+    public readonly uuidInstance: ClassInstantiation;
+
+    constructor({ value }: UUID.Args) {
+        super();
+        this.uuidInstance = this.generateUuidInstance(value);
+    }
+
+    public write(writer: Writer): void {
+        this.uuidInstance.write(writer);
+    }
+
+    private generateUuidInstance(value: string): ClassInstantiation {
+        const uuidInstance = new ClassInstantiation({
+            classReference: new Reference({
+                name: "UUID",
+                modulePath: ["uuid"]
+            }),
+            arguments_: [
+                new MethodArgument({
+                    value: new CodeBlock(`"${value}"`)
+                })
+            ]
+        });
+
+        this.inheritReferences(uuidInstance);
+        return uuidInstance;
+    }
+}
```

Comment:
My understanding of `Type` is that it's used to render a type hint. Such as `my_var: List[str]`. The goal of this ast node is to render an instance of a UUID, such as `my_var = UUID("<some-uuid>")`. Given this context, do you still feel that way? If so, am I thinking of `Type` incorrectly?

Reply:
Got it, so in this case it would normally be a `TypeInstantiation` similar ([C#](https://github.com/fern-api/fern/blob/27e99c941c244e45e69f4f728ad50c70c410b00e/generators/csharp/codegen/src/ast/InstantiatedPrimitive.ts#L134) and [Go](https://github.com/fern-api/fern/blob/27e99c941c244e45e69f4f728ad50c70c410b00e/generators/go-v2/codegen/src/ast/TypeInstantiation.ts#L172)).

It's not the end of the world to include this here as-is, but I'd be wary of creating separate AST nodes for other primitive type instantiations like this since mainly because it's inconsistent.

Reply:
Ah okay, I'll close this PR and introduce a separate `TypeInstantiation` AstNode.